otj-server-reactive/src/main/java/com/opentable/server/reactive/logging/ReactiveServerLoggingConfiguration.java

```diff
@@ -24,6 +24,7 @@
 /**
  * Configures automatic server request logging using Jetty for reactive applications.
  */
```

Comment:
Add an `@deprecated` segment to the JavaDoc, so people know what is supposed to be done instead

Reply:
👍